contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -4724,6 +4724,20 @@ static bool is_valid_set_option(std::string val)
 		(pg_strcasecmp("QUERY_GOVERNOR_COST_LIMIT", val.c_str()) == 0);
 }
 
+static PLtsql_variable * build_babelfish_guc_variable(const char* val)
+{
+	int type = InvalidOid;
+	if (pg_strcasecmp("ROWCOUNT", val) == 0)
+		type = INT4OID;
+
+	if (type == InvalidOid)
+		return NULL;
+	
+	PLtsql_variable *var = pltsql_build_variable(psprintf("babelfishpg_tsql.%s", val), 0, pltsql_build_datatype(type, -1, InvalidOid, NULL), false);
```

Comment:
Is there any other variable go the same way ?

Reply:
There are some other variable, I need to check all the allowed once.